README.md

```diff
@@ -245,6 +245,37 @@ For example: `{"token":"*YOUR_CONNECTION_PASSWORD*", "kodiid":"bedroom"}`
 </details>
 
 
+### **B.4) [OPTIONAL] Set up a local instance using Docker**
+As an alternative to (B.2), it's possible to use a pre-build [Docker image](https://hub.docker.com/r/sinedied/googlehomekodi/) to run a local instance.
```

Comment:
what architectures are you supporting? i am trying to get arm32v6 and arm32v7 running to support raspberries pis